common/OpTestSSH.py

```diff
@@ -155,52 +159,131 @@ def get_console(self):
 
         return self.console
 
-    def run_command(self, command, timeout=60):
-        console = self.get_console()
-        if self.prompt:
-            prompt = self.prompt
+    def set_env(self, console):
+        set_env_list = []
+        console.sendline("exec bash --norc --noprofile")
```

Comment:
actually bash is not there in petitboot shell, so this will fail when in skiroot tests. So can you keep it like this.
```
if self.use_default_bash:
            console.sendline("exec bash --norc --noprofile")
```